src/main/java/ua/danit/final_project/entities/Schedule.java

```diff
@@ -0,0 +1,31 @@
+package ua.danit.final_project.entities;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "schedule")
+@Data
+public class Schedule implements Serializable {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @ManyToOne(fetch = FetchType.LAZY, optional = false)
+  @JoinColumn(name = "r_id", nullable = false)
+  private Role role;
+
+  private Timestamp start;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "start" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=59&issues=AWUy2H-hW2AOP0Ck_lbR&open=AWUy2H-hW2AOP0Ck_lbR)

[ISSUE_KEY]: AWUy2H-hW2AOP0Ck_lbR